src/js/react/components/ChatInterface.jsx

```
@@ -101,49 +107,105 @@ function Answer({ md }) {
-        reaction === 'upvote'
-          ? 'Thanks for the feedback!'
-          : 'Feedback received',
-        'success'
-      )
-    } catch (err) {
-      console.error('Feedback error', err)
-      showToast('Could not send feedback', 'error')
+// ——— Sources ————————————————————————————————————————————————————————————
+// Assistant messages carry sources inside tool-call blocks; collect and
+// dedupe them by URL for a compact link list under the answer.
+function extractSources(blocks) {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)